spec.md

```diff
@@ -2315,6 +2419,18 @@ message NodeServiceCapability {
       GET_VOLUME_STATS = 2;
       // See VolumeExpansion for details.
       EXPAND_VOLUME = 3;
+      // Indicates the SP supports node level volume condition.
```

Comment:
ditto re: "node level" - can we find better terminology than "controller/node-level"? maybe it's confusing for me because CSI plugin arch is flexible and so "level" doesn't really make sense for e.g. monolith plugins that combine both gRPC services.

suggestion: "Indicates that the Node service can report volume conditions."

Reply:
Done